resources/language/resource.language.es_es/strings.po

```diff
@@ -926,7 +930,7 @@ msgstr "Instalar complemento Up Next"
 
 msgctxt "#30235"
 msgid "[WIP] Synchronize the watched status of the videos with Netflix"
-msgstr "[En desarrollo] Sincronizar el estado de visto de los vídeos con Netflix"
+msgstr "[WIP] Sincronizar el estado de visto de los vídeos con Netflix"
```

Comment:
As far I know 'WIP' means 'Work In Progress'. I think it's better to translate this acronym because very few people know its actual meaning.

Reply:
I know what WIP is I always try to make the user interface look as clean as possible.
I do not see wrong using WIP besides this way we remove some characters from the line option in the menu.
if someone doesn't know you can always use the device next to it called mobile and google what WIP stands for

Reply:
> I know what WIP is I always try to make the user interface look as clean as possible.
> I do not see wrong using WIP besides this way we remove some characters from the line option in the menu.
> if someone doesn't know you can always use the device next to it called mobile and google what WIP stands for

But are we translating the entire text or only a part of it? In fact, 'Work In Progress' has an exact translation in our language, if not I would agree with you. It means 'trabajo en elaboración' according to the previous online dictionary:
https://www.wordreference.com/es/translation.asp?tranword=work%20in%20progress

However, you're right, this text could not fit well in the dialog box. So what about '[En pruebas]' or even '[Experimental]''? Since it seems this feature is not well tested yet. And believe me, the average user does not know what 'WIP' means. All users are not developers or translators, or they don't have the specific knowledge to know the meaning of 'WIP'. We should make the user's life easier in my opinion (that is, to avoid searching this term in Google with a mobile phone).

Reply:
I could change the WIP text in english with another text
I was looking for an abbreviated formula to take up little space

At first I was thinking of using [TEST]
but then i opted for [WIP] because i thought i could finish it,
but it's getting complex and long to do

it might help to change english WIP with TEST?

Reply:
I think that the correct is leave WIP. Many add-ons use it.